tensorrt_llm/_torch/auto_deploy/custom_ops/fused_moe/trtllm_moe.py

```
@@ -235,4 +237,110 @@ def trtllm_quant_fp8_moe_fused_fake(
+        quant_scales=quant_scales,
+        input_sf=input_sf,
+        activation_type=activation_type,
+    )[0].view(x.shape)
+
+    return trtllm_output[0].view(x.shape)
+
+
+@trtllm_quant_nvfp4_moe_fused.register_fake
+def trtllm_quant_nvfp4_moe_fused_fake(
+    hidden_states: torch.Tensor,
+    router_weight: torch.Tensor,
```

Ruff (0.14.6)

Unused function argument: `router_weight`

(ARG001)